test/e2e/framework/util.go

```diff
@@ -335,10 +316,7 @@ func CreateTestingNS(baseName string, c clientset.Interface, labels map[string]s
 	// We don't use ObjectMeta.GenerateName feature, as in case of API call
 	// failure we don't know whether the namespace was created and what is its
 	// name.
-	name, err := findAvailableNamespaceName(baseName, c)
-	if err != nil {
-		return nil, err
-	}
+	name := fmt.Sprintf("%v-%v", baseName, RandomSuffix())
```

Comment:
the old code actually tries to find an avail. name for 30 seconds with 2 second intervals in a loop, so ~15 retires.
this seems OK.

with this change however, we stop doing that an regenerate on conflict a single time in CreateTestingNS.
in my opinion the old code is more robust.

Reply:
the problem of old code is before the actual creation, the name may be taken by other processes.

my fix is to regenerate the name in `wait.PollImmediate(Poll, 30*time.Second, func() (bool, error) {`, try a new name each time.

Reply:
right, i missed that the creation is already in a loop.

Reply:
thank you for the explanation.